websites/management/commands/markdown_cleaning/rules/broken_link_fix.py

```diff
@@ -48,7 +48,7 @@ class BrokenLinkFixRuleMixin(ABC):
             - /pages/page2
     """
 
-    Parser = partial(LinkParser, recursive=True)
```

Comment:
Some context for this change on https://github.com/mitodl/ocw-studio/issues/2650#issuecomment-3201418763.